packages/resin-universal-gosquared/src/node.js

```diff
@@ -0,0 +1,38 @@
+var Promise = require('bluebird')
+var GoSquared = require('gosquared')
+
+module.exports = function(gosquaredId, apiKey, debug) {
+	var visitor = null
+	function createVisitor(userId) {
+		if (visitor) return visitor
+		var gosquared = new GoSquared({
+			api_key: apiKey,
+			site_token: gosquaredId
+		})
+
+		if (userId) {
+			gosquared.createPerson(userId)
```

Comment:
Yea, if you don't identify a user with `.createPerson()` the lib will handle it for you. So it think it's simpler to only assign a user if an id is provided. 

See the on this example on this [page](https://www.gosquared.com/docs/api/tracking/event/node/) it should clarify.